### ISSUE TITLE
Issue with JESD3-C files containing test vectors

### ISSUE THREAD

**Issue by roscopeco** [roscopeco/atfprog-tools#4]
**Workaround**: Do not include test vectors (in WinCUPL this means not using the `.si` or disabling WinSIM).

Certain `.jed` files output by WinCUPL (with WinSIM) contain apparently-invalid test vectors, which causes the 
programming software to fail to parse the file. 

**These vectors should be completely ignored by the programmer.**

Attached JED file: 

```
(.venv) ➜  new_glue git:(bringup) atfu program -d ATF1508 xosera_glue.jed
Traceback (most recent call last):
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/.venv/bin/atfu", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/src/atfu/command.py", line 15, in main
    args.func(args)
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/src/atfu/program.py", line 21, in handler
    xsvf_files = _process_input_files(args.device, args.filename, output)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/src/atfu/program.py", line 31, in _process_input_files
    return [_process_input_file(device, result_file, output) for result_file in files]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/src/atfu/program.py", line 31, in <listcomp>
    return [_process_input_file(device, result_file, output) for result_file in files]
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/src/atfu/program.py", line 36, in _process_input_file
    return _process_input_svf(output, _process_input_jed(output, device, file))
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/src/atfu/program.py", line 51, in _process_input_jed
    return jed2svf(device=device, infile=file, output=output)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/src/atfu/converter/jed2svf.py", line 35, in jed2svf
    jed_bits, comment = read_jed(infile)
                        ^^^^^^^^^^^^^^^^
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/src/atfu/converter/fuseconv.py", line 10, in read_jed
    parser.parse()
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/src/atfu/converter/jesd3.py", line 160, in parse
    for token, position, args in self._lexer:
  File "/Users/rosco/Development/electronics/atf15xxprog/atfprog-tools/src/atfu/converter/jesd3.py", line 109, in __next__
    raise JESD3ParsingError(
atfu.converter.jesd3.JESD3ParsingError: field V has invalid format at line 913, column 3
```

[bug3.zip](https://github.com/roscopeco/atfprog-tools/files/15439430/bug3.zip)

**Comment by roscopeco** [comment]
[bug3-actual.zip](https://github.com/roscopeco/atfprog-tools/files/15439450/bug3-actual.zip)

**Comment by roscopeco** [comment]
Fixed and release 2024.1013
